src/client.js

```diff
@@ -9,9 +9,22 @@ import { template, templateStyles } from './app/lib/joinUsTemplate';
 const data = window.SIMORGH_DATA || {};
 const root = document.getElementById('root');
 
-Loadable.preloadReady().then(() => {
-  hydrate(<ClientApp data={data} routes={routes} />, root);
-});
+// Only hydrate the client if we're on the expected path
+// When on an unknown route, the SSR would be discarded and the user would only
+// see a blank screen. Avoid this by only hydrating when the embedded page data
+// and window location agree what the path is. Otherwise, fallback to the SSR.
+if (window.SIMORGH_DATA.path === window.location.pathname) {
+  Loadable.preloadReady().then(() => {
+    hydrate(<ClientApp data={data} routes={routes} />, root);
+  });
+} else {
+  // eslint-disable-next-line no-console
+  console.log(`
```

Comment:
Soz to be a pain, can you use `logger.node` please? it has a client swap in that will log for us

Reply:
:+1: